nameko_twilio.py

```diff
@@ -0,0 +1,18 @@
+from nameko.extensions import DependencyProvider
+from twilio.rest import Client
+
+
+class Twilio(DependencyProvider):
+
+    def setup(self):
+        self.account_sid = self.container.config['account_sid']
+        self.auth_token = self.container.config['auth_token']
```

Comment:
let's rename the config variables to `twilio_account_sid` and `twilio_auth_token`
What if I was to have another dependency in my nameko app that had an auth token and an account id? This would be confusing.
`self.account_sid` and `self.auth_token` are fine though because they belong to the `Twilio` class